### PR TITLE
predicates/primitive: document False predicates ignores arguments

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
This feature could be used to a add comment why route has it.

### CODE REVIEW

**Comment by AlexanderYastrebov** [comment]
True() also ignored its arguments but I don't thing we should document it.

**Comment by MustafaSaber** [comment]
👍

File: docs/reference/predicates.md [hunk 1]
```diff
@@ -229,13 +229,13 @@ route2: Path("/test") && True() -> "http://www.github.com";
...
 
-Does not match. Can be used to disable certain routes.
+Does not match. Can be used to disable certain routes. Ignores all arguments.
 
 Example where `route2` is disabled.
 
 ```
 route1: Path("/test") -> "http://www.zalando.de";
-route2: Path("/test") && False() -> "http://www.github.com";
+route2: Path("/test") && False("Disabled due to maintenance") -> "http://www.github.com";
```
Comment (szuecs) [inline docs/reference/predicates.md:238]: Why?
I wonder why you relax input to a predicate which normally you would rather be strict about. :)

**Comment by szuecs** [comment]
:-1:

**Comment by AlexanderYastrebov** [comment]
I think its fine to allow a comment for False predicate but this comment can also be a part of route sourse e.g. eskip file or RouteGroup yaml. Lets come to this idea later if we see the need.

**Pull request closed without merging**